cluster/src/main/java/org/apache/iotdb/cluster/client/rpcutils/TElasticFramedTransport.java

```diff
@@ -26,6 +26,13 @@
 
 public class TElasticFramedTransport extends TFastFramedTransport {
 
+  /**
+   * It is used to prevent the size of the parsing package from being too large and allotting the
```

Comment:
allotting -> allocating, which is more universal

Reply:
Thanks